### DISCUSSION TITLE
Jumbo MTU

### DISCUSSION THREAD

**Discussion by ccy** [ipxe/ipxe#691] [Q&A]
I configure option 26 (MTU) in my DHCP service to `9000`.

After I boot into iPXE, the MTU of the network interface show `9000`:

```
iPXE> show net0/ip
net0.dhcp/ip:ipv4 = 192.168.0.186

iPXE> show net0/mtu
net0.dhcp/mtu:int16 = 9000
```
I did not get any response when try to ping a jumbo packet with **prohibit fragmentation** to the iPXE host:

    ping -M do -s 8972 192.168.0.186

However, ping with standard MTU (1500) do return response:

    ping -M do -s 1472 192.168.0.186

Just wondering Is iPXE network work with jumbo MTU?

**Comment by NiKiZe** [comment]
Some drivers have support, others do not.
Please check ifstat and post output.

If the driver is UNDI there is not much hope. For SNP or NII there is a maybe but depends on the underlying firmware and driver.

If you find issues with a specific iPXE native driver, feel free to open issue for that specific driver.

**Reply by ccy** [reply]
Even though the MTU setting is 9000 in iPXE host and jumbo ping without fragmentation isn't work properly, but it seems transferring large files in a 9000 MTU network environment work too.

**Reply by NiKiZe** [reply]
Again, which driver? What does ifstat in iPXE show?

**Reply by ccy** [reply]
I am using **snponly.efi** driver.  Here is the `ifstat` output:

```
iPRE> ifstat
net0: 00:15:5d:00:c2:06 using SNP on SNP-Ox7f9fbf98 (Ethernet) [open]
  [Link:up, TK:21 THE:2 RX:1498 RXE:1226]
  [TKE: 1 x "Network unreachable (https://ipxe.org/28006090)"]
  [TXE: 1 x "Error 0x2a006089 (https://ipxe.org/2a006089)]
  [RXE: 64 x "Operation not supported (https://ipxe.org/3c046083)]
  [RXE: 991 x "The socket is not connected (https://ipxe.org/380f6093)]
  [RXE: 165 x "Operation not supported (https://ipxe.org/3c006083)]
  [RXE: 1 x "Invalid argument (https://ipxe.org/1c056082)]
```

**Reply by NiKiZe** [reply]
Then you are in the hands of your RFI firmware. Try an upgrade and verify with manufacturer.
